cmd/notification.go

```diff
@@ -114,6 +114,9 @@ func (sys *NotificationSys) DeletePolicy(policyName string) []NotificationPeerEr
 		}
 		client := client
 		ng.Go(GlobalContext, func() error {
+			if !client.IsOnline() {
```

Comment:
This should be done inside peer-rest-client.go not sure why we need it here.

Reply:
moving it into peer-rest-client.go won't solve the problem. ng.Go has 3 in built re-tries, causing multiple reload attempts just because one peer node is down.

Reply:
Added the check in peer-rest-client as well for the benefit of other callers.

Reply:
We should change the retries configurable @poornas